packages/dd-trace/src/config.js

```diff
@@ -133,3 +164,5 @@ function isFalse (str) {
 }
 
 module.exports = Config
+module.exports.isTrue = isTrue
```

Comment:
These are utilities unrelated to the configuration, so if they are exporter they should be moved to another file.

Reply:
any idea where they should be moved to? i don't think we have a "general utils" file(?)